components/providers/provider-card.tsx

```
@@ -1,71 +1,119 @@
+import Link from 'next/link';
+import { useState } from 'react';
 
 interface ProviderCardProps {
   id: number;
   name: string;
-  heroImageUrl?: string | null;
+  heroImageUrl: string;
   aboutSnippet?: string;
-  about?: string;
-  categories?: Category[];
+  categories?: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)